public/counter-pick.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "id": "aa_heavy",
+    "title": "AA-heavy",
+    "description": "Blind them, or make sure you can compensate their damage (they don't have burst). Then punish them.",
+    "features": [
+      "blind",
```

Comment:
I would use an object like in heroes.
It's more flexible and I guarantee you if app is gonna grow it will really help.

Reply:
Well, it's easier to do now than later. It's not like code is gonna get much bigger

Reply:
Ok, if you think it's better for the project - i don't mind